tests/resources/service-templates/tosca-simple-1.0/node-cellar/node-cellar.yaml

```diff
@@ -302,6 +306,14 @@ topology_template:
     capabilities:
       app_endpoint: [ loadbalancer, client ]
 
+  outputs:
```

Comment:
It'd be better (or rather, also good) to have an `outputs` section appended to the `hello-world` example, and then use it in the end2end test to grab the URL of the web server.